docs/sources/_index.md

```diff
@@ -8,6 +8,8 @@ menuTitle: Grafana Cloud k6
 description: Run k6 in Cloud servers, visualize and scripts in your Grafana instance.
 description: 'Script performance tests locally, run tests from the CLI on servers all over the world, and visualize test results in Grafana Cloud.'
 weight: 001
+aliases:
+  - /docs/k6
```

Comment:
nit: I think the true URL is `/docs/k6/` because of Hugo pretty URLs. I think we rewrite the path to include the trailing slash anyway so I'm not sure this makes a difference.

Reply:
fixed by https://github.com/grafana/k6-docs/pull/1088/commits/7fd963e2234656db958422ab50532b2a46c82c01